library_generation/test/model/library_config_unit_tests.py

```diff
@@ -37,3 +38,29 @@ def test_get_library_returns_api_shortname(self):
             gapic_configs=list(),
         )
         self.assertEqual("secret", library.get_library_name())
+
+    def test_get_sorted_gapic_configs_returns_correct_order(self):
```

Comment:
Can we create a test for each of the scenario? [v1alpha1, v1], [v1, v2] etc. Because 
1. It would be much more readable. For example, just by reading the tests, I don't know why `v1alpha1, v1, v2, admin_v2, non_versioned, v1beta1` is sorted to `v2, v1, admin_v2, v1beta1, v1alpha1, non_versioned`. 
2. We can also easily test more scenarios. e.g. scenarios of two non versioned libraries.

Reply:
I added unit tests to verify comparison result of two `gapic_config`s.